app/models/admin_set.rb

```diff
@@ -83,13 +83,35 @@ def active_workflow
   # Calculate and update who should have edit access based on who
   # has "manage" access in the PermissionTemplateAccess
   def reset_access_controls!
-    # NOTE: This is different from Collections in that it doesn't update read access.  See the notes in services/hyrax/collections/permissions_service.rb
-    update!(edit_users: permission_template.agent_ids_for(access: 'manage', agent_type: 'user'),
-            edit_groups: permission_template.agent_ids_for(access: 'manage', agent_type: 'group'))
+    # NOTE: This is different from Collections in that it doesn't update read access based on visibility.
+    #       See also app/models/concerns/hyrax/collection_behavior.rb#reset_access_controls!
+    update!(edit_users: permission_template_edit_users,
+            edit_groups: permission_template_edit_groups,
+            read_users: permission_template_read_users,
+            read_groups: permission_template_read_groups)
```

Comment:
Add read access for depositors and viewers.  Excludes groups `['registered', 'public']`